contracts/Oracle.sol

```diff
@@ -28,6 +28,13 @@ contract Oracle is Ownable {
     /// @dev mapping between batch and it price at specific timestmap. A batch is the hash of underlying, collateral, strike and expiry.
     mapping(bytes32 => mapping(uint256 => Price)) internal batchPriceAt;
 
+    /// @notice emits an event when an oracle added for a specific batch
+    event BatchOracleAdded(bytes32 indexed batch, address oracle);
+    /// @notice emits an event when a locking period added for a specific oracle
+    event OracleLockingPeriodAdded(address indexed oracle, uint256 lockingPeriod);
+    /// @notice emits an event when a dispute period added for a specific oracle
+    event OracleDisputePeriodAdded(address indexed oracle, uint256 disputePeriod);
```

Comment:
Maybe `BatchOracleUpdated ` and `OracleLockingPeriodUpdated `?

Reply:
cool updated this.